docs/services/MicrosoftTranslator.md

```diff
@@ -24,13 +24,14 @@ The **Microsoft Translator Service** allows you to translate text to various sup
 await TranslatorService.Instance.InitializeAsync("<translator service key");
 
 // Retrieves friendly names for the languages available for text translation.
-var languages = await TranslatorService.Instance.GetLanguageNamesAsync();
+var languages = await TranslatorService.Instance.GetLanguageNamesAsync("en");
```

Comment:
The docs are getting removed from this repo. Could you update the docs here instead: https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs

Reply:
Yes, no problem :-)